week1/index.js

```diff
@@ -1 +1,57 @@
-console.log("Hello world")
\ No newline at end of file
+import HTTP from 'http'
+import dflt from './resFun/dflt.js'
+import addFn from './resFun/addFn.js'
+import removeFn from './resFun/removeFn.js'
+import resetFn from './resFun/resetFn.js'
+// import Path from 'path'
+
+var state = 10
+let addOne = () => ++state
+let reState = () => state = 10
+let subOne = () => --state
+
+const server = HTTP.createServer((request, response)=>{
+    console.log(request.method, request.url)
+    switch (request.url){
+        case '/':
+        dflt(response)
+        break
+        case '/state':
+        response.write(state.toString())
+        break
+        case '/add':
+        addOne()
+        addFn(response)
+        response.write(state.toString())
```

Comment:
Why do you call both `addFn` which writes the response, and `response.write` with `state` directly?